mui/mui.py

```diff
@@ -234,6 +167,42 @@ def edit_custom_hook(bv: BinaryView, addr: int):
             bv.session_data.mui_custom_hooks[addr] = dialog.text()
 
 
+def load_evm(bv: BinaryView):
+    filename = get_open_filename_input("filename:", "*.sol").decode()
+    if filename is None:
+        return
+
+    print(filename)
+    filename = Path(filename)
+
+    os.chdir(filename.parent.resolve())
+    output = CryticCompile(
+        filename.name,
+        solc_solcs_bin=settings.get_string(f"{BINJA_EVM_RUN_SETTINGS_PREFIX}solc_path", bv),
+    )
```

Comment:
Is this `os.chdir` necessary? Is it something to do with `CryticCompile`?

Reply:
Yes, it's a workaround to prevent `CryticCompile` errors. I added a comment to clarify that.